docs/client-options.md

```diff
@@ -48,6 +48,10 @@ third party clients.
 - [hlandau/acme](https://github.com/hlandau/acme)
 - [ericchiang/letsencrypt](https://github.com/ericchiang/letsencrypt)
 
+## Java
```

Comment:
Could you move your new Java client section to below the HAProxy section to preserve the alphabetic order?